src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,53 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+interface Props {
+  onChangeFilter: (event: React.ChangeEvent<HTMLSelectElement>) => void,
+  query: string,
+  onChangeQuery: (event: React.ChangeEvent<HTMLInputElement>) => void,
+  onClearQuery: () => void,
+}
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+export const TodoFilter: React.FC<Props> = ({
+  onChangeFilter,
+  query,
+  onChangeQuery,
+  onClearQuery,
+}) => {
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select data-cy="statusSelect" onChange={onChangeFilter}>
+            <option value="all">All</option>
+            <option value="active">Active</option>
+            <option value="completed">Completed</option>
```

Comment:
Create enum with all filters and use it here 
```suggestion
            <option value={FilterBy.All}>All</option>
            <option value={FilterBy.Active}>Active</option>
            <option value={FilterBy.Completed}>Completed</option>
```